website-components-playground/gatsby-node.js

```diff
@@ -31,3 +31,21 @@ exports.onCreateBabelConfig = ({ actions }) => {
     },
   });
 };
+
+exports.createPages = ({ actions }) => {
+  const { createRedirect } = actions;
+
+  createRedirect({
+    fromPath: `/tabular-modal-page/`,
+    toPath: `/tabular-modal-page/tab-one`,
+    redirectInBrowser: true,
+    isPermanent: true,
+  });
+
+  createRedirect({
+    fromPath: `/tabular-modal-page`,
+    toPath: `/tabular-modal-page/tab-one`,
+    redirectInBrowser: true,
+    isPermanent: true,
+  });
```

Comment:
This is strange, but `react-router-dom`'s `<Redirect />` doesn't seem to be working in Gatsby (at least I couldn't make it work). That is how I solved. There is a Gatsby plugin to trail the ending slash, but I wasn't sure if this solution is acceptable at all so I didn't add that package at this point